plaso/parsers/winreg.py

```diff
@@ -170,23 +198,25 @@ def _ParseRecurseKeys(self, parser_mediator, root_key):
       if parser_mediator.abort:
         break
 
-      matching_plugin = None
+      self._ParseKey(parser_mediator, registry_key)
```

Comment:
what is the value of replacing this method with another method?

Reply:
I build _ParseKey to avoid duplication of the same code in _ParseRecurseKeys and _ParseKeysFromFindSepcs.

Reply:
ack